src/components/Hero/HeroSearchBar/HeroSearchBar.jsx

```diff
@@ -0,0 +1,63 @@
+import { useCallback, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { route } from '@/constants';
+import { AppSearchInput, AppSelect } from '@/shared';
+import {
+  HeroSearchBarFormStyled,
+  HeroSearchButtonStyles,
+} from './HeroSearchBar.styled';
+
+const searchType = Object.freeze({
+  CHEF: 'Chef',
+  DISH: 'Dish',
+});
+
+const HeroSearchBar = () => {
+  const navigate = useNavigate();
+  const [type, setType] = useState(searchType.DISH);
+  const [query, setQuery] = useState('');
+
+  const changeTypeHandler = useCallback((evt) => {
+    setType(evt.target.value);
+  }, []);
+  const changeQueryHandler = useCallback((evt) => {
+    setQuery(evt.target.value);
+  }, []);
+  const submitHandler = useCallback(
+    (evt) => {
+      evt.preventDefault();
+      const searchQuery = query.trim();
+
```

Comment:
If we don't need to perform a search with empty parameters and display all dishes or all chefs, should we return a message and not navigate to the next page?

Reply:
If search is empty, we show all dishes or chefs.